### PR TITLE
Clickhole medal tracker

### PR DESCRIPTION
**Pull request by daytonn**

### CODE REVIEW

**Comment by MichaelButkovic** [comment]
LGTM

File: elements/clickhole-medal-tracker/clickhole-medal-tracker-examples.js [hunk 1]
```diff
@@ -9,6 +9,27 @@ let examples = {
...
+            src="http://localhost:8080/fixtures/clickhole-medal-tracker/medals.json"
+            disable-animation>
+          </clickhole-medal-tracker>`;
+      },
+    },
+    'ClickholeMedalTracker (animation options)': {
+      render () {
+        return `
+          <clickhole-medal-tracker
+            src="http://localhost:8080/fixtures/clickhole-medal-tracker/medals.json"
```
Comment (shawncook) [inline elements/clickhole-medal-tracker/clickhole-medal-tracker-examples.js]: what up with this

Comment (shawncook) [inline elements/clickhole-medal-tracker/clickhole-medal-tracker-examples.js]: never mind

**Comment by collin** [comment]
👍 Stop those crazy animations.

**Pull request merged**